docs/versioning/versioning-overview.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+nav_order: 2
+title: Versioning Overview 
+has_children: false
+parent: Versioning
+---
+# Versioning Overview 
+
+Provena provides capability for versioning of a relevant subset of entity types in the Registry, including datasets. The purpose of this feature is to allow users to create and manage versions of registered items in Provena. Users can version items to mark items at a point in their development or release lifecycle. Users can then manage and refer to versions of an item. This can then provide a more accurate provenance trail, e.g. version 1 of Dataset X uses a different configuration of a modelling software suite to version 2 of Dataset X.
+
+Other use cases include:
+* Marking a previous version of an item as deprecated or erroneous
+* Maintaining two versions of items that are valid but have different lineages
+
+
+## What can be versioned?
+
+* Datasets
+* Model
+* Dataset template 
+* Model run workflow template
+
+## What cannot be versioned?
+
+* Agents - people, organisations
+* Model workflow run records 
+
+## What happens when an item is versioned
+
+1. A new persistent identifier is minted for each version (i.e. a new Handle ID given)
+2. A versioned item is timestamped
+3. Versions are tracked via a provenance graph. A version event is captured as a "Version activity"
+4. A version is given a version number, which is incremented by 1 each time a subsequent version is created
+5. Versioned items shall link to the previous revision. Users should be able to navigate back to other versions (previous and later versions)
+6. A versioned item is a separate entity to other versioned items. Users should be able to view a versioned item independently of previous versions
+7. Provena provides functionality to navigate to the current revision and previous revisions distinctly
```

Comment:
These are not steps - these are axioms i.e. "The versioned item _shall_ refer to the previous version". 

These don't belong in a sequential list, but might still be important?

Reply:
you're right, this wasn't meant to be steps. they are a list of things that happen when a version is created.